### PR TITLE
My work seccion

### PR DESCRIPTION
**Pull request by Sirvelasque**
Here I put my work section with the space and samples where my projects will be show with a picture, summary and tools info, also I added a buton for more details and replicated 3 times in a 4 items 1 column grid

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -75,9 +74,75 @@ <h2>Software developer</h2>
...
+          <div class="work_img"></div>
+          <h3 class="works_titles">Multi-Post Stories</h3>
+          <p class="project_st">A daily selection of privately personalized reads;no accounts or sign-ups required. has been the industry's 
+             standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <div class="work_tools_tags">
+            <div class="tool_tag">css</div>
+            <div class="tool_tag">html</div>
+            <div class="tool_tag">bootstrap</div>
+            <div class="tool_tag">Ruby</div>
+          </div>
```
Comment (rotimiazeez) [inline html/index.html]: - [ ] Please note that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li` 😉

File: html/index.html [hunk 2]
```diff
@@ -75,9 +74,75 @@ <h2>Software developer</h2>
...
+            <button type="button" class="project_btn">See project</button>
+        </div>
+      </div>
+    </section>
+
```
Comment (rotimiazeez) [inline html/index.html:136]: - [ ] Kindly make sure the about section is added to complete this milestone👍

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Sirvelasque 👍, 
## Good job so far!
### Summary 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: html/index.html (hunk 2 above)
Comment (Sirvelasque) [reply html/index.html:136]: hello @rotimiazeez ! thank you for the review, can I have one more hint? I mean, the template doesn't show an about section so I would like to have an idea about how to solve this

File: html/index.html [hunk 3]
```diff
@@ -75,9 +74,75 @@ <h2>Software developer</h2>
...
+          <div class="work_img"></div>
+          <h3 class="works_titles">Multi-Post Stories</h3>
+          <p class="project_st">A daily selection of privately personalized reads;no accounts or sign-ups required. has been the industry's 
+             standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <ul class="work_tools_tags">
+            <li class="tool_tag">css</li>
+            <li class="tool_tag">html</li>
+            <li class="tool_tag">bootstrap</li>
+            <li class="tool_tag">Ruby</li>
+          </ul>
```
Comment (mateo951) [inline html/index.html:93]: Addressed 👌🔥

File: html/index.html (hunk 2 above)
Comment (mateo951) [reply html/index.html:136]: @Sirvelasque yes you are right 👀 There is no about me section on your template 🤯

image

**Review by mateo951 — APPROVED** [review]
Hi @Sirvelasque, 

Great job making the adjustments required by your previous reviewer 🔥
Your project is complete!  There is nothing else to say other than... it's time to merge it 😎 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**